conf/prod.routes

```diff
@@ -1,6 +1,6 @@
 ->          /                             health.Routes
 ->          /                             dynamicDefinition.Routes
 
-GET         /status-request/mrn/:mrn      @uk.gov.hmrc.customs.declarations.information.controllers.InformationController.get(mrn)
+GET         /status-request/mrn/:mrn/status      @uk.gov.hmrc.customs.declarations.information.controllers.InformationController.get(mrn)
```

Comment:
There's not meant to be `status-request` in the path! That affects everything else in this PR so I'll review when it's all fixed.